addon/server.js

```diff
@@ -474,7 +598,8 @@ export default class Server {
     let routeHandler = new RouteHandler({
       schema: this.schema,
       verb, rawHandler, customizedCode, options, path,
-      serializerOrRegistry: this.serializerOrRegistry
+      serializerOrRegistry: this.serializerOrRegistry,
+      server: this
```

Comment:
I'm still not seeing it... doesn't `server.js` persist the db by calling `savePersistenceDump()` from the `handledRequest` hook?